server/index.ts

```diff
@@ -0,0 +1,32 @@
+import 'module-alias/register';
+import express from 'express';
+import morgan from 'morgan';
+import bodyParser from 'body-parser';
```

Comment:
You don't need `bodyParser` it already built-in with `express.json()` here the [docs](https://expressjs.com/en/api.html#express.json)